src/_includes/docs/learning-resources-index/grid.liquid

```diff
@@ -0,0 +1,72 @@
+<section class="card-grid" id="all-resources-grid">
+    {%- for item in resources -%}
+        <a id="{{ item.name }}"
+           data-type="{{ item.type }}"
+           data-tags='{{ item.tags | join: ", " }}'
+           data-description="{{ item.description }}"
+           class="card outlined-card"
+           href="{{ item.link.url }}"
+           target="_blank"
+        >
+            <div class="card-image-holder-material-3">
+                {%- if item.imageUrl -%}
+                    <img src='{{ item.imageUrl }}'>
+                {%- endif -%}
+            </div>
```

Comment:
It looks like you're not using this, but if you intend to, consider moving the `if` outside of the parent `div` so it's only rendered if needed.

Reply:
Good catch. I do intend to add images over time, so I'll keep this in (with your suggested fix)